src/sentry/models/event.py

```diff
@@ -58,16 +57,6 @@ class EventCommon(object):
     Methods and properties common to both Event and SnubaEvent.
     """
 
-    @classmethod
-    def generate_node_id(cls, project_id, event_id):
```

Comment:
Could this be used in plugins that are still not in the sentry repository?

Reply:
Just double checked, it's not used in any other repository